src/allotropy/parsers/beckman_vi_cell_xr/vi_cell_xr_reader.py

```diff
@@ -113,50 +123,34 @@ def _get_file_version(self) -> XrVersion:
 
 
 class ViCellXRTXTReader:
-    data: pd.DataFrame
+    data: list[SeriesData]
     series: SeriesData
 
     def __init__(self, contents: NamedFileContents) -> None:
         self.lines = read_to_lines(contents)
         self.file_info = self._get_file_info()
         self.data = self._read_data()
 
-    def _read_data(self) -> pd.DataFrame:
-        # generate long, single column dataframe
-        file_data = pd.DataFrame(data=StringIO("\n".join(self.lines)))
-
-        # strip whitespace
-        file_data[0] = file_data[0].str.strip()
-
-        # drop empty cells
-        file_data = file_data.replace("", None).dropna(axis=0, how="all")
-
-        # split on colon surrounded by any amount of space, limited to one split to avoid destroying datetimes
-        file_data = file_data[file_data.columns[0]].str.split(
-            r"\s*:\s*", n=1, expand=True
+    def _read_data(self) -> list[SeriesData]:
+        data_frame = read_csv(
+            StringIO("\n".join(self.lines)), sep=" :", index_col=0, engine="python"
         )
-
-        # pivot the long data of a single sample to a df that is 1 wide row
-        file_data["Pivot Index"] = 1
-        file_data = file_data.pivot(index="Pivot Index", columns=0)
-
-        # rename the columns to match the existing parser that was based on xls(x) files
-        file_data.columns = pd.Index(
-            [
-                HEADINGS_TO_PARSER_HEADINGS.get(name, name)
-                for name in [x[1] for x in file_data.columns]
-            ]
+        file_data = df_to_series(
```

Comment:
Note: the crux of the fix. Instead of splitting the dataframe up and pivoting, use the `:` separator in the first place (with a leading space to preserve dates) and then just transpose it.